### PR TITLE
p4-1114 add disabled_at to prison_transfer_reasons entity

### PR DESCRIPTION
**Pull request by starswan**
Fixes P4-1114

## Proposed Changes

- Adds disabled_at to prison transfer reason

## To test/demo change

- call prison_transfer_reasons endpoint and see disabled_at property for a disabled entity

## Things to check & link
- [X] API docs has been updated if necessary

### CODE REVIEW

File: swagger/v1/prison_transfer_reason.json [hunk 1]
```diff
@@ -35,6 +35,19 @@
...
+              {
+                "type": "string",
+                "format": "date-time"
+              },
+              {
+                "type": "null"
+              }
+            ],
+            "example": "2017-07-21T17:32:28Z",
+            "description": "The date-time at which a value was disabled, in ISO-8601 format, or null if it is enabled"
```
Comment (alexdesi) [inline swagger/v1/prison_transfer_reason.json]: I do not get the meaning in the description, it sounds quite generic "which a value was disabled",
the value of what?

**Comment by martyn-w** [comment]
how about using the [Discard](https://github.com/jhawthorn/discard) gem which we have used on other models (see e.g. Document, Subscription, Notification)

File: swagger/v1/prison_transfer_reason.json (hunk 1 above)
Comment (starswan) [reply swagger/v1/prison_transfer_reason.json]: This is copied from the other fields with a same name

**Comment by starswan** [comment]
> how about using the [Discard](https://github.com/jhawthorn/discard) gem which we have used on other models (see e.g. Document, Subscription, Notification)

The semantics of this are completely different. This is a field exposed to the client for it to work with - it is on all our reference data.

**Comment by martyn-w** [comment]
> > how about using the [Discard](https://github.com/jhawthorn/discard) gem which we have used on other models (see e.g. Document, Subscription, Notification)
> 
> The semantics of this are completely different. This is a field exposed to the client for it to work with - it is on all our reference data.

Sorry, I misunderstood the use-case 👍

**Pull request merged**